api/v1alpha1/bundle_types.go

```diff
@@ -95,6 +100,16 @@ type GitRef struct {
 	Commit string `json:"commit,omitempty"`
 }
 
+type Authorization struct {
+	// Type defines the kind of authorization method being used.
+	AuthorizationType string `json:"authorizationType,omitempty"`
+	// Secret contains reference to the secret that has authorization information for HTTPS protocol and is in the namespace that the provisioner is deployed.
+	// The secret is expected to contain `data.username` and `data.password` for the username and password, respectively.
+	Secret corev1.LocalObjectReference `json:"secret,omitempty"`
+	// InsecureSkipVerify is set true, the server certificate is ignored. It is not safe to set this true for production environment use.
```

Comment:
```suggestion
	// InsecureSkipVerify controls whether a client verifies the server's certificate chain and host name. This
	// configuration is used only when cloning git repositories that use the "https" scheme. If InsecureSkipVerify
	// is true, the clone operation will accept any certificate presented by the server and any host name in that
	// certificate. In this mode, TLS is susceptible to machine-in-the-middle attacks unless custom verification is
	// used. This should be used only for testing.
```

(I re-used a bunch from the upstream GoDoc about in the `crypto/tls` package.)